static/app/components/quickTrace/index.tsx

```diff
@@ -86,8 +88,52 @@ export default function QuickTrace({
   const traceLength = quickTrace.trace && quickTrace.trace.length;
   const {root, ancestors, parent, children, descendants, current} = parsedQuickTrace;
 
+  const hasReplay =
+    Boolean(event.entries?.find(({type}) => type === 'breadcrumbs')) &&
+    Boolean(event.tags?.find(({key}) => key === 'replayId')?.value);
```

Comment:
Totally fine, but IMO would be clearer as:
```suggestion
  const hasReplay =
    event.entries?.some(({type}) => type === 'breadcrumbs') &&
    event.tags?.some(({key, value}) => key === 'replayId' && Boolean(value));
```